app/models/usuario.rb

```diff
@@ -0,0 +1,3 @@
+class Usuario < ApplicationRecord
```

Comment:
Please use English for the code base, so rename Usuario as User and the table columns accordingly